app/components/mandaat/burgemeester-selector.js

```diff
@@ -196,4 +210,19 @@ export default class MandaatBurgemeesterSelectorComponent extends Component {
   get toolTipText() {
     return this.persoon ? 'Er is reeds een burgemeester geselecteerd.' : '';
   }
+
+  @action
+  closeEditBurgemeesterModal() {
+    this.editBurgemeester = false;
+    this.isEditFormInitialized = false;
+  }
+
+  @action
+  async saveBurgemeesterChanges({ instanceId }) {
+    await this.fractieApi.updateCurrentFractie(instanceId);
+    this.closeEditBurgemeesterModal();
+    if (this.args.onUpdateBurgemeester) {
+      setTimeout(() => this.args.onUpdateBurgemeester(), 1000);
+    }
```

Comment:
This should trigger a router refresh, but it does not seem to be happening, anybody got an idea why?

Reply:
Added a comment in the RC chat for this for using the draft mandataris component togheter with the IV service